test/mitmproxy/net/data/verificationcerts/generate.py

```diff
@@ -92,3 +92,22 @@ def mkcert(cert, subject, ip: bool):
         pem.write(key.read())
 
 shutil.copyfile("trusted-leaf.pem", "example.mitmproxy.org.pem")
+with (
```

Comment:
Should we also test if the file has an intermediate cert but no leaf cert?

Reply:
Good call, added!